src/Network/Network/ChangeLog.md

```diff
@@ -35,6 +35,7 @@
 * Added support for new ErGWScale SKU - ErGwScale
 * Added property 'size' to firewallPolicy and firewallPolicyRuleCollectionGroup.
 * Updated cmdlet `New-AzBastion`, `Set-AzBastion` and `Get-AzBastion` to support Bastion features for CRUD operations
+* Added optional property 'Profile' to `New-AzFirewallPolicyIntrusionDetection` 
```

Comment:
Change log should be added in upcoming release.

Reply:
thank you, fixed